cwms-data-api/src/test/resources/cwms/cda/data/dto/property.json

```diff
@@ -0,0 +1,6 @@
+{
+  "category": "TestCategory",
+  "name": "TestName",
+  "office": "TestOffice",
```

Comment:
lots of cda stuff is office-id and not office

Reply:
confusing that the query parameter is office and the DTO is office-id.
What about "name"? I see a similar mismatch and inconsistency among DTO's there